worlds/tloz/Locations.py

```diff
@@ -0,0 +1,350 @@
+        "Level 5 Key Drop (Gibdos Entrance)", "Level 5 Key Drop (Gibdos, Keese, and Pols Voice)",
+        "Level 5 Key Drop (Zols)", "Level 5 Bomb Drop (Gibdos)",
+        "Level 5 Bomb Drop (Dodongos)", "Level 5 Rupee Drop (Zols)",
+    ],
+    [
+        "Level 6 Item (Magical Rod)", "Level 6 Map", "Level 6 Compass", "Level 6 Boss", "Level 6 Triforce",
+        "Level 6 Key Drop (Wizzrobes Entrance)", "Level 6 Key Drop (Keese)",
+        "Level 6 Key Drop (Wizzrobes North Island)", "Level 6 Key Drop (Wizzrobes North Stream)",
+        "Level 6 Key Drop (Vires)", "Level 6 Bomb Drop (Wizzrobes)",
+        "Level 6 Rupee Drop (Wizzrobes)"
+    ],
+    [
+        "Level 7 Item (Red Candle)", "Level 7 Map", "Level 7 Compass", "Level 7 Boss", "Level 7 Triforce",
+        "Level 7 Key Drop (Ropes)", "Level 7 Key Drop (Goriyas)", "Level 7 Key Drop (Stalfos)",
+        "Level 7 Key Drop (Moldorms)", "Level 7 Bomb Drop (Goriyas South)", "Level 7 Bomb Drop (Keese and Spikes)",
+        "Level 7 Bomb Drop (Moldorms South)", "Level 7 Bomb Drop (Moldorms North)",
+        "Level 7 Bomb Drop (Goriyas North)", "Level 7 Bomb Drop (Dodongos)",
+        "Level 7 Bomb Drop (Digdogger)", "Level 7 Rupee Drop (Goriyas Central)",
+        "Level 7 Rupee Drop (Dodongos)", "Level 7 Rupee Drop (Goriyas North)",
+    ],
+    [
+        "Level 8 Item (Magical Key)", "Level 8 Map", "Level 8 Compass", "Level 8 Item (Book of Magic)", "Level 8 Boss",
+        "Level 8 Triforce", "Level 8 Key Drop (Darknuts West)",
+        "Level 8 Key Drop (Darknuts Far West)", "Level 8 Key Drop (Pols Voice South)",
+        "Level 8 Key Drop (Pols Voice and Keese)", "Level 8 Key Drop (Darknuts Central)",
+        "Level 8 Key Drop (Keese and Zols Entrance)", "Level 8 Bomb Drop (Darknuts North)",
+        "Level 8 Bomb Drop (Darknuts East)", "Level 8 Bomb Drop (Pols Voice North)",
+        "Level 8 Rupee Drop (Manhandla Entrance West)", "Level 8 Rupee Drop (Manhandla Entrance North)",
+        "Level 8 Rupee Drop (Darknuts and Gibdos)",
+    ],
+    [
+        "Level 9 Item (Silver Arrow)", "Level 9 Item (Red Ring)",
+        "Level 9 Map", "Level 9 Compass",
+        "Level 9 Key Drop (Patra Southwest)", "Level 9 Key Drop (Like Likes and Zols East)",
+        "Level 9 Key Drop (Wizzrobes and Bubbles East)", "Level 9 Key Drop (Wizzrobes East Island)",
+        "Level 9 Bomb Drop (Blue Lanmolas)", "Level 9 Bomb Drop (Gels Lake)",
+        "Level 9 Bomb Drop (Like Likes and Zols Corridor)", "Level 9 Bomb Drop (Patra Northeast)",
+        "Level 9 Bomb Drop (Vires)", "Level 9 Rupee Drop (Wizzrobes West Island)",
+        "Level 9 Rupee Drop (Red Lanmolas)", "Level 9 Rupee Drop (Keese Southwest)",
+        "Level 9 Rupee Drop (Keese Central Island))", "Level 9 Rupee Drop (Wizzrobes Central)",
+        "Level 9 Rupee Drop (Wizzrobes North Island)", "Level 9 Rupee Drop (Gels East)"
+    ]
+]
+
+all_level_locations = []
```

Comment:
`all_level_locations = [location for level in level_locations for location in level]